taxi/urls.py

```diff
@@ -37,14 +42,35 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("cars/", CarListView.as_view(),
+         name="car-list"),
+    path("cars/<int:pk>/", CarDetailView.as_view(),
+         name="car-detail"),
+    path("cars/create/", CarCreateView.as_view(),
+         name="car-create"),
+    path("cars/<int:pk>/update/", CarUpdateView.as_view(),
+         name="car-update"),
+    path("cars/<int:pk>/delete/", CarDeleteView.as_view(),
+         name="car-delete"),
+    path("drivers/", DriverListView.as_view(),
+         name="driver-list"),
+    path("drivers/<int:pk>/", DriverDetailView.as_view(),
+         name="driver-detail"),
+    path("drivers/create/", DriverCreateView.as_view(),
+         name="driver-create"),
+    path("drivers/<int:pk>/delete/", DriverDeleteView.as_view(),
+         name="driver-delete"),
+    path("drivers/<int:pk>/update/", DriverUpdateView.as_view(),
+         name="driver-update"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/<int:car_id>/assign-driver/",
+        AssignDriverToCarView.as_view(),
+        name="assign-driver",
+    ),
+    path(
+        "cars/<int:car_id>/drivers/<int:driver_id>/remove/",
+        RemoveDriverFromCarView.as_view(),
+        name="remove-driver",
```

Comment:
The same issue here, you should use `-` to split words in URL identification parameter `name`, not the `_`. So, `remove-driver` should be `remove_driver`.